src/raster/rasterband.rs

```diff
@@ -0,0 +1,160 @@
+use libc::{c_int, c_void};
+use raster::{gdal, Dataset, Buffer};
+use raster::types::{GdalType};
+use raster::gdal_enums;
+use gdal_major_object::MajorObject;
+use metadata::Metadata;
+
+pub struct RasterBand<'a> {
+    c_rasterband: *const c_void,
+    owning_dataset: &'a Dataset,
+}
+
+impl <'a> RasterBand<'a> {
+    pub fn owning_dataset(&self) -> &'a Dataset {
+        self.owning_dataset
+    }
+
+    pub unsafe fn _with_c_ptr(c_rasterband: *const c_void, owning_dataset: &'a Dataset) -> Self {

```

Comment:
I followed the example of the other structs like `Dataset`. Should we change this globally?